docs/sdk/pnp/web/no-modal/initialize.mdx

```diff
@@ -59,36 +58,87 @@ This Web3AuthNoModal constructor takes an object with `Web3AuthNoModalOptions` a
 
 - `Object`: The web3auth instance with all its methods and events.
 
+### Instantiating a Provider
+
+<InstantiatingProvider />
+
+### Whitelabeling
+
+#### `uiConfig`
+
+For customizing the redirect screens while logging in and constructing the key, you need to pass on `WhiteLabelData` in the `uiConfig` parameter to
+the `uiConfig` property of Web3Auth.
+
+:::tip
+
+This is just one of the aspects of whitelabeling you can achieve with Web3Auth. To know more in-depth about how you can Whitelabel your application
+with Web3Auth, have a look at our [Whitelabeling SDK Reference](/pnp/features/whitelabel).
+
+:::
+
+##### Example
+
+```tsx
+const uiConfig: WhiteLabelData = {
+  appName: "My App",
+  appURL: "https://example.com",
+  logoLight: "https://example.com/logo-light.png",
+  logoDark: "https://example.com/logo-dark.png",
+  defaultLanguage: "en",
+  mode: "light",
+  useLogoLoader: true,
+  theme: {
+    primary: "#FF0000",
+    secondary: "#00FF00",
+    warning: "#FFA500",
+  },
+  tncLink: "https://example.com/terms",
+  privacyPolicy: "https://example.com/privacy",
+};
+
+const web3auth = new Web3AuthNoModal({
+  clientId: "", // Get your Client ID from the Web3Auth Dashboard
+  chainConfig,
+  web3AuthNetwork: "sapphire_mainnet",
```

Comment:
You can change text "sapphire_mainnet" to type web3AuthNetwork: WEB3AUTH_NETWORK.SAPPHIRE_MAINNET in examples
import { WEB3AUTH_NETWORK } from "@web3auth/base";

Reply:
Done